New hosts UI features (#1736)
theforeman/foreman-documentation: guides/common/modules/proc_changing-the-environment-of-a-host.adoc

.Procedure
. In the {ProjectWebUI}, navigate to *Hosts* > *All hosts*.

> . Click the name of the hosts you want to modify.

Reviewer: ```suggestion
. Click the name of the host you want to modify.
```

> . Click the vertical ellipsis icon next to the *Content view details* card, then clicking *Edit content view assignment*.

Reviewer: ```suggestion
. Click the vertical ellipsis in the *Content view details* card and select *Edit content view assignment*.
```
